go/teams/member_test.go

```diff
@@ -529,20 +523,20 @@ func TestMemberListInviteUsername(t *testing.T) {
 	})
 
 	annotatedTeamList, err := List(context.TODO(), tc.G, keybase1.TeamListArg{UserAssertion: "", All: true})
-	if err != nil {
-		t.Fatal(err)
-	}
-	if len(annotatedTeamList.AnnotatedActiveInvites) != 1 {
-		t.Fatalf("active invites: %d, expected 1", len(annotatedTeamList.AnnotatedActiveInvites))
-	}
-	for _, invite := range annotatedTeamList.AnnotatedActiveInvites {
-		if invite.TeamName != name {
-			t.Errorf("invite team name: %q, expected %q", invite.TeamName, name)
-		}
-		if string(invite.Name) != username {
-			t.Errorf("invite username: %q, expected %q", invite.Name, username)
+	require.NoError(t, err)
+	require.Equal(t, 0, len(annotatedTeamList.AnnotatedActiveInvites))
+	require.Equal(t, 2, len(annotatedTeamList.Teams))
+
+	var foundMember bool
+	for _, member := range annotatedTeamList.Teams {
```

Comment:
I'm confused what `annotatedTeamList.Teams` is.  Sounds like a list of teams, but this loop makes it look like a list of members?

Reply:
It's a design flaw that's been there since the beginning. teamList with `All: true` argument will return members in `Teams` field. I didn't want to change the API right now